index.html

```diff
@@ -509,12 +493,16 @@ <h2>
         realm</a></dfn> is defined in [[!ECMASCRIPT]].
       </p>
       <p>
-        The terms <a href=
+        The terms and concepts <a href=
         "https://dom.spec.whatwg.org/#eventtarget"><dfn><code>EventTarget</code></dfn></a>,
         <a href=
         "https://dom.spec.whatwg.org/#event"><dfn><code>Event</code></dfn></a>,
         <a href=
-        "https://dom.spec.whatwg.org/#dictdef-eventinit"><dfn><code>EventInit</code></dfn></a>
+        "https://dom.spec.whatwg.org/#dictdef-eventinit"><dfn><code>EventInit</code></dfn></a>,
+        and <dfn data-lt="fire|fires an event|fire an event"><a href=
```

Comment:
I think the "and" should go before "trusted event."

Reply:
Thanks, now fixed.